backend/core/socketio_config.py

```diff
@@ -317,33 +403,33 @@ async def client_to_user(body: str):
         session = Session(engine)
         message = json.loads(body)
 
-        user_id, user_phone_number = handle_incoming_message(
-            session=session, message=message
+        user_id, user_phone_number, chat_session_id, chat_id = (
+            handle_incoming_message(session=session, message=message)
         )
+        user_sid = get_cache(user_id=user_id)
 
         conversation_envelope = get_value_or_raise_error(
             message, "conversation_envelope"
         )
         conversation_envelope.update({"user_phone_number": user_phone_number})
+        conversation_envelope.update({"chat_session_id": chat_session_id})
+        conversation_envelope.update({"message_id": chat_id})
         message.pop("conversation_envelope", None)
         message.update({"conversation_envelope": conversation_envelope})
 
-        user_sid = get_cache(user_id=user_id)
-
-        logger.info(
-            f"Send client->user to {USER_CACHE_KEY}{user_id} "
-            f"{user_sid}: {message}"
+        # send message to user_chats
+        await rabbitmq_client.producer(
+            body=body,
+            routing_key=RABBITMQ_QUEUE_USER_CHATS,
         )
-
         await sio_server.emit(
             "chats", message, to=user_sid, callback=emit_chats_callback
         )
-
-        await rabbitmq_client.initialize()
-        await rabbitmq_client.producer(
-            body=body,
-            routing_key=RABBITMQ_QUEUE_USER_CHATS,
+        logger.info(
+            f"Send client->user to {USER_CACHE_KEY}{user_id} "
+            f"{user_sid}: {message}"
         )
+
     except Exception as e:
         logger.error(f"Error handling user_chats_callback: {e}")
```

Comment:
This message should probably read `client_to_user`.

Reply:
Sorry, I didn't get that.

Reply:
`user_chats_callback` is a left-over from renaming the function from `user_chats_callback` to `client_to_user`. This message still uses the old name, it seems. Maybe I missed that when I renamed things.